network/vcn/subnets.tf

```diff
@@ -195,3 +195,60 @@ resource "oci_core_subnet" "k8sWorkerSubnetAD3" {
     command = "sleep 5"
   }
 }
+
+
+##### LB Subnets
+
+resource "oci_core_subnet" "k8sCCMLBSubnetAD1" {
+  availability_domain        = "${lookup(data.oci_identity_availability_domains.ADs.availability_domains[0],"name")}"
+  cidr_block                 = "10.0.50.0/24"
+  compartment_id             = "${var.compartment_ocid}"
+  display_name               = "${var.label_prefix}${var.control_plane_subnet_access}K8SCCMLBSubnetAD1"
+  dns_label                  = "k8sccmlbad1"
+  vcn_id                     = "${oci_core_virtual_network.CompleteVCN.id}"
+  route_table_id             = "${var.control_plane_subnet_access == "private" ? coalesce(join(" ", oci_core_route_table.NATInstanceAD1RouteTable.*.id), join(" ", oci_core_route_table.NATInstanceAD2RouteTable.*.id), join(" ", oci_core_route_table.NATInstanceAD3RouteTable.*.id), oci_core_route_table.PublicRouteTable.id) : oci_core_route_table.PublicRouteTable.id}"
+  dhcp_options_id            = "${oci_core_virtual_network.CompleteVCN.default_dhcp_options_id}"
+  security_list_ids          = ["${oci_core_security_list.K8SCCMLBSubnet.id}"]
+  prohibit_public_ip_on_vnic = "${var.control_plane_subnet_access == "private" ? "true" : "false"}"
```

Comment:
The subnets created for the CCM should not be allowed to ever be private - unless we _also_ expect the CCM to create private load balancers for services, which it doesn't. 

So, I think that instead of queuing off `control_plane_subnet_access`,  `prohibit_public_ip_on_vnic` should always be `false`.

Reply:
And if we're not going to allow the CCM subnets to be private:

`display_name  = "${var.label_prefix}PublicK8SCCMLBSubnetADX"`

and:

`route_table_id      = "${oci_core_route_table.PublicRouteTable.id}"`